README.md

```diff
@@ -1,16 +1,7 @@
 # Probabilistic Programming
 ## Scottish Programming Languages and Verification Summer School 2019
-
+### Running
 ```
-$ cabal install Chart
-$ cabal install Chart-diagrams
-$ cabal install Chart-cairo
-$ cabal install data-default
-$ cabal install logfloat
-$ cabal install erf
-$ cabal install transformers
-$ cabal install cassava
-$ cabal install pretty-show
-$ ghci Casino.hs
-*Casino> main
+stack ghci
```

Comment:
Should this say `stack ghci Casino.hs` ?

Reply:
You could do it, but all the modules are available to you once you do `stack ghci`, so you could instead do just `import Casino`

Reply:
I just forgot to write this in!